lib/mongo/events.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Mongo.Events.ServerDescriptionChangedEvent do
```

Comment:
Another idea is to make all `GenEvent.notify(Mongo.Events, ...)` API on `Mongo.Events` and never expose the structs/records outside of the `Mongo.Events` module.